python/README.rst

```diff
@@ -103,10 +103,31 @@ For Mac OSX
     # or for python3 (you may have to uninstall boost and reinstall to build python3 libs)
     $ brew install boost-python3
 
-Also, having Anaconda in your path can cause segmentation faults when importing the pyvw module. Providing Conda support
-is an open issue and efforts are welcome, but in the meantime it is suggested to remove any conda bin directory from your path
-prior to installing the vowpalwabbit package.
-
+Having `conda` on a Linux-based system requires a build with conda-provided packages to ensure consistency:
+.. code-block:: bash
+    # create/activate a conda environment (depends on your choice, no example is given)
+    # boost packages
+    conda install boost libboost py-boost
+    # a consistent compiler (g++ 7.2.0)
+    conda install gxx_linux-64
+    # pick up the newly installed compiler
+    source activate YourEnvironmentName
```

Comment:
Can `YourEnvironmentName` be automatically determined inside conda?

Reply:
Yes, will be done in the proposed script